cmd/collector/main.go

```diff
@@ -73,10 +71,6 @@ func main() {
 			}
 			logger := svc.Logger // shortcut
 			baseFactory := svc.MetricsFactory.Namespace(metrics.NSOptions{Name: "jaeger"})
```

Comment:
```suggestion
			metricsFactory := svc.MetricsFactory.Namespace(metrics.NSOptions{Name: "jaeger"})
```
(unless baseFactory is used anywhere else)